CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.2.5
+* Update DETELE endpoint to take customer types into account [#734](https://github.com/stellar/java-stellar-anchor-sdk/pull/734)
+
```

Comment:
I would always phrase the change from the perspective of the developer using the service/library. This is a bug fix so lets frame it as such instead of an update.

We should also get in the habit of structuring our changelog for better readability. See an [example](https://github.com/stellar/django-polaris/releases/tag/v2.3.0) of how I organized Django Polaris's changelog.

```suggestion
## 1.2.5
* Fix `DELETE /customer` request by checking all configured customer types [#734](https://github.com/stellar/java-stellar-anchor-sdk/pull/734)

```